From 93ed4909a336fdd2a9f56ec95aaa5ac9004e94e5 Mon Sep 17 00:00:00 2001
From: Igor Grinberg <igrinberg@habana.ai>
Date: Sun, 28 Oct 2018 20:14:54 +0200
Subject: [PATCH] uboot: goya/gaudi: consolidate the env files

There are several identical settings inside six environment files.
Consolidate them into one file in a common location.

This leaves Gaudi files empty, but do not delete them instead, leave
them as a place holders for future changes.

Signed-off-by: Igor Grinberg <igrinberg@habana.ai>
---

diff --git a/board/habanalabs/common/uboot-env-common.txt b/board/habanalabs/common/uboot-env-common.txt
new file mode 100644
index 0000000..c35886a
--- /dev/null
+++ b/board/habanalabs/common/uboot-env-common.txt
@@ -0,0 +1,2 @@
+bootm_low=0
+bootcmd=while true; do waitcmd; run command; done
diff --git a/board/habanalabs/gaudi/uboot-env-asic.txt b/board/habanalabs/gaudi/uboot-env-asic.txt
index c35886a..e69de29 100644
--- a/board/habanalabs/gaudi/uboot-env-asic.txt
+++ b/board/habanalabs/gaudi/uboot-env-asic.txt
@@ -1,2 +0,0 @@
-bootm_low=0
-bootcmd=while true; do waitcmd; run command; done
diff --git a/board/habanalabs/gaudi/uboot-env-onix.txt b/board/habanalabs/gaudi/uboot-env-onix.txt
index c35886a..e69de29 100644
--- a/board/habanalabs/gaudi/uboot-env-onix.txt
+++ b/board/habanalabs/gaudi/uboot-env-onix.txt
@@ -1,2 +0,0 @@
-bootm_low=0
-bootcmd=while true; do waitcmd; run command; done
diff --git a/board/habanalabs/gaudi/uboot-env-pldm.txt b/board/habanalabs/gaudi/uboot-env-pldm.txt
index c35886a..e69de29 100644
--- a/board/habanalabs/gaudi/uboot-env-pldm.txt
+++ b/board/habanalabs/gaudi/uboot-env-pldm.txt
@@ -1,2 +0,0 @@
-bootm_low=0
-bootcmd=while true; do waitcmd; run command; done
diff --git a/board/habanalabs/goya/uboot-env-asic.txt b/board/habanalabs/goya/uboot-env-asic.txt
index 57b9192..eed6438 100644
--- a/board/habanalabs/goya/uboot-env-asic.txt
+++ b/board/habanalabs/goya/uboot-env-asic.txt
@@ -1,4 +1,2 @@
-bootm_low=0
 ethaddr=00:33:66:99:cc:ff
 netboot=dhcp 192.168.10.109:goya-fit.itb && bootm $fileaddr#conf@1
-bootcmd=while true; do waitcmd; run command; done
diff --git a/board/habanalabs/goya/uboot-env-onix.txt b/board/habanalabs/goya/uboot-env-onix.txt
index c823bf6..d437049 100644
--- a/board/habanalabs/goya/uboot-env-onix.txt
+++ b/board/habanalabs/goya/uboot-env-onix.txt
@@ -1,4 +1,2 @@
-bootm_low=0
 ethaddr=00:11:22:33:44:55
 netboot=dhcp 192.168.10.86:goya-fit.itb && bootm $fileaddr#conf@1
-bootcmd=while true; do waitcmd; run command; done
diff --git a/board/habanalabs/goya/uboot-env-pldm.txt b/board/habanalabs/goya/uboot-env-pldm.txt
index 7649fbf..c03176d 100644
--- a/board/habanalabs/goya/uboot-env-pldm.txt
+++ b/board/habanalabs/goya/uboot-env-pldm.txt
@@ -1,3 +1 @@
-bootm_low=0
 ethaddr=00:22:44:66:88:aa
-bootcmd=while true; do waitcmd; run command; done
diff --git a/configs/gaudi_defconfig b/configs/gaudi_defconfig
index d5ec6fb..f4ca90f 100644
--- a/configs/gaudi_defconfig
+++ b/configs/gaudi_defconfig
@@ -53,7 +53,7 @@
 BR2_TARGET_UBOOT_NEEDS_DTC=y
 BR2_TARGET_UBOOT_SPL=y
 BR2_TARGET_UBOOT_ENVIMAGE=y
-BR2_TARGET_UBOOT_ENVIMAGE_SOURCE="$(BR2_EXTERNAL_HABANALABS_PATH)/board/habanalabs/$(HL_DEV)/uboot-env-$(HL_TARGET).txt"
+BR2_TARGET_UBOOT_ENVIMAGE_SOURCE="$(BR2_EXTERNAL_HABANALABS_PATH)/board/habanalabs/common/uboot-env-common.txt $(BR2_EXTERNAL_HABANALABS_PATH)/board/habanalabs/$(HL_DEV)/uboot-env-$(HL_TARGET).txt"
 BR2_TARGET_UBOOT_ENVIMAGE_SIZE="0x1000"
 BR2_TARGET_UBOOT_ENVIMAGE_REDUNDANT=y
 BR2_PACKAGE_HOST_UBOOT_TOOLS=y
diff --git a/configs/goya_defconfig b/configs/goya_defconfig
index f9690d0..731eb8b 100644
--- a/configs/goya_defconfig
+++ b/configs/goya_defconfig
@@ -53,7 +53,7 @@
 BR2_TARGET_UBOOT_NEEDS_DTC=y
 BR2_TARGET_UBOOT_SPL=y
 BR2_TARGET_UBOOT_ENVIMAGE=y
-BR2_TARGET_UBOOT_ENVIMAGE_SOURCE="$(BR2_EXTERNAL_HABANALABS_PATH)/board/habanalabs/$(HL_DEV)/uboot-env-$(HL_TARGET).txt"
+BR2_TARGET_UBOOT_ENVIMAGE_SOURCE="$(BR2_EXTERNAL_HABANALABS_PATH)/board/habanalabs/common/uboot-env-common.txt $(BR2_EXTERNAL_HABANALABS_PATH)/board/habanalabs/$(HL_DEV)/uboot-env-$(HL_TARGET).txt"
 BR2_TARGET_UBOOT_ENVIMAGE_SIZE="0x1000"
 BR2_TARGET_UBOOT_ENVIMAGE_REDUNDANT=y
 BR2_PACKAGE_HOST_UBOOT_TOOLS=y
